packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -388,6 +388,10 @@ export class Position {
         }
         return false;
     }
+
+    toJSON(): any {
+		return { line: this.line, character: this.character };
+	}
```

Comment:
Please enable eslint (or run the linter defined in the `package.json` with `yarn lint`). You used `tab` characters here - instead of spaces - which are discouraged by our eslint config. The same also applies to the other change.

Reply:
fixed